### ISSUE TITLE
idea for websocket/webrtc streaming

### ISSUE THREAD

**Issue by 1ucay** [quiniouben/vban#27]
Hello, it would be possible on web page listen audio, which is sync on multiple devices?

**Comment by quiniouben** [comment]
Hi, do I understand correctly that you would like a GUI (possibly a web-page based on websocket) to select and listen to different streams ?

**Comment by 1ucay** [comment]
Yes exactly. What I want is listen audio without "VBAN Receptor" APP. Only JS solution with websocket or webrtc.

**Comment by quiniouben** [comment]
Ok, then unfortunately it has few chance to happen in this project soon, except if someone contributes. Two main reasons:

- I am no web technology developer. I have some little knowledge but I don't develop anything with these ;
- to my little knowledge, and I may be completely wrong, there is no standard universal way (I mean working in all browsers) to open udp ports in JS code. If this keypoint would be solved, then, the protocol is very simple.

**Comment by 1ucay** [comment]
Thank you. In some free time
Websocket using TCP, so no possible. But WebRTC has datachannel https://www.html5rocks.com/en/tutorials/webrtc/datachannels/
OR maybe via node.js
https://nodejs.org/api/dgram.html#dgram_udp_datagram_sockets

What do you think? Can this work?
